blerpc/src/main/java/com/blerpc/BleRpcController.java

```diff
@@ -58,4 +58,11 @@ public boolean isCanceled() {
     public void notifyOnCancel(RpcCallback<Object> callback) {
         throw new UnsupportedOperationException("Not implemented.");
     }
+
+    /**
+     * Asks that the given callback be called when the subscription was succeed. The callback will
```

Comment:
```
A callback that is called when a subscription process finished successfully.

It will always be called exactly once for {@link MethodType#SUBSCRIBE} methods.
```